packages/dataclass-editor/src/master/useValidation.ts

```diff
@@ -0,0 +1,7 @@
+import { useAppContext } from '../context/AppContext';
+import type { DataClassField } from '../data/dataclass';
+
+export const useValidation = (field: DataClassField) => {
+  const { validationMessages } = useAppContext();
+  return validationMessages.filter(message => message.path === field.name);
```

Comment:
maybe you can also think about to directly convert this validation messages to MessageData objects, which you then directly can use for the messageRows and also for the BasicFields...

Reply:
Done in the next PR: https://github.com/axonivy/dataclass-editor-client/pull/45